app/src/main/java/com/github/orkest/View/auth/SignUpForm.kt

```diff
@@ -154,6 +157,51 @@ fun SignUpForm(navController: NavController, viewModel: AuthViewModel) {
                         )
                     }
                 }
+
```

Comment:
I don't think this part should be in the same UI as the sign Up page, we should keep the UIs for sign up and for sign in separated

Reply:
Yes you're right, I've changed it